integration/tests/access/access_api_test.go

```diff
@@ -193,6 +194,14 @@ func (s *AccessAPISuite) testGetAccount(client *client.Client) {
 		s.Assert().Equal(serviceAddress, account.Address)
 		s.Assert().NotZero(serviceAddress, account.Balance)
 	})
+
+	s.Run("get newly created account", func() {
+		addr, err := utils.CreateFlowAccount(s.ctx, s.serviceClient)
+		s.Require().NoError(err)
+		acc, err := client.GetAccount(s.ctx, addr)
+		s.Require().NoError(err)
+		s.Assert().Equal(addr, acc.Address)
```

Comment:
maybe also check that the balance is greater than 0

Reply:
717f24c1abb714f981a0149ea450fe54d6a9511b